bin/lib/checkout.sh

```diff
@@ -80,28 +80,9 @@ function checkout::get_image_tag() {
 #######################################
 function checkout::from_image_tag() {
   local image_tag="${1}"
-
-  if snapshots::tag_is_snapshot "${image_tag}"; then
-    checkout::from_snapshot "${image_tag}"
-  elif [[ "${image_tag}" = "latest" ]]; then
-    printf "Setting checkout to latest... "
-    checkout::at_main > /dev/null
-    echo "done"
-  else
-    out::error "Only SNAPSHOT tags and 'latest' are currently supported."
-    exit 1
+  if ! commit_sha=$(exec bin/lib/resolve_git_commit_sha_from_image.py --tag="${image_tag}"); then
+    exit $?
```

Comment:
it would be great to have an option to checkout latest from git directly, instead of latest image sha.

Reply:
I assume this is something we'd want only for debugging (e.g. as we fix the deployment code, we'd need to test the changes). I added a USE_LATEST_GIT_COMMIT environment variable which bypasses all of this logic.